src/service/cli/server.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const chalk = require(`chalk`);
+const http = require(`http`);
+const fs = require(`fs`).promises;
+const {HttpCode} = require(`../../constants`);
+
+const DEFAULT_PORT = 3000;
+const FILE_NAME = `mocks.json`;
+
+const sendResponse = (res, statusCode, message) => {
+  const template = `
+    <!Doctype html>
+      <html lang="ru">
+      <head>
+        <title>With love from Node</title>
+      </head>
+      <body>${message}</body>
+    </html>`.trim();
+
+  res.writeHead(statusCode, {
+    'Content-Type': `text/html; charset=UTF-8`,
+  });
+
+  res.end(template);
+};
+
+const onClientConnect = async (req, res) => {
+  const notFoundMessageText = `Not found`;
+
+  switch (req.url) {
+
+    case `/`:
+      try {
+        const fileContent = await fs.readFile(FILE_NAME);
+        const mocks = JSON.parse(fileContent);
+        const message = mocks.map((post) => `<li>${post.title}</li>`).join(``);
+        sendResponse(res, HttpCode.OK, `<ul>${message}</ul>`);
+      } catch (err) {
+        sendResponse(res, HttpCode.NOT_FOUND, notFoundMessageText);
+      }
+
+      break;
```

Comment:
```suggestion
    case `/`: {
      try {
        const fileContent = await fs.readFile(FILE_NAME);
        const mocks = JSON.parse(fileContent);
        const message = mocks.map((post) => `<li>${post.title}</li>`).join(``);
        sendResponse(res, HttpCode.OK, `<ul>${message}</ul>`);
      } catch (err) {
        sendResponse(res, HttpCode.NOT_FOUND, notFoundMessageText);
      }

      break;
      }
```
во всех кейсах должны быть курли